pkg/abstractions/image/build.go

```diff
@@ -451,7 +456,7 @@ func (b *Builder) generatePipInstallCommand(pythonPackages []string, pythonVersi
 		}
 	}
 
-	command := fmt.Sprintf("%s -m pip install --root-user-action=ignore", pythonVersion)
+	command := "uv pip install --system --link-mode=copy"
```

Comment:
Can you copy/paste what this looks like when building a container?

Reply:
It looks like this:
```
=> Building image
Waiting for build container to start...
Python 3.10.15
6
Using Python 3.10.15 environment at /usr
Resolved 35 packages in 434ms
Prepared 28 packages in 3.17s

Saving image, this may take a few minutes...
Installed 28 packages in 238ms
 + certifi==2024.8.30
 + contourpy==1.3.0
 + cycler==0.12.1
 + filelock==3.16.1
 + fonttools==4.54.1
 + fsspec==2024.10.0
 + huggingface-hub==0.26.1
 + idna==3.10
 + joblib==1.4.2
 + kiwisolver==1.4.7
 + matplotlib==3.9.2
 + numpy==2.1.2
 + opencv-python==4.10.0.84
 + pandas==2.2.3
 + pillow==11.0.0
 + pytz==2024.2
 + pyyaml==6.0.2
 + regex==2024.9.11
 + safetensors==0.4.5
 + scikit-learn==1.5.2
 + scipy==1.14.1
 + seaborn==0.13.2
 + threadpoolctl==3.5.0
 + tokenizers==0.20.1
 + tqdm==4.66.5
 + transformers==4.46.0
 + tzdata==2024.2
 + urllib3==1.26.20
=> Build complete 🎉
=> Syncing files
```

Reply:
Oh great - its still readable. I haven't run uv yet, was mostly curious.